bigml/api.py

```diff
@@ -362,6 +375,74 @@ def __init__(self, username=None, api_key=None,
             self.listers[resource_type] = getattr(self,
                                                   "list_%s" % method_name)
 
+    def create(self, resource_type, *args, **kwargs):
+        """Create resources
+
+        """
+        finished = kwargs.get('finished', True)
+        create_kwargs = filter_kwargs(kwargs,
+                                      ['query_string', 'finished'],
+                                      out=True)
+        try:
+            resource_info = self.creaters[resource_type](*args,
+                                                         **create_kwargs)
```

Comment:
i'd call them creators, creaters sounds weird

Reply:
oh, that's been there for some time...